Add empty example dashboards to ddev create templates

nmuesch opened the pull request:
### What does this PR do?
<!-- A brief description of the change being made with this pull request. -->
Adds a new folder `assets->dashboards` to the templates for new integrations 

### Motivation
<!-- What inspired you to submit this pull request? -->
Create a full scaffolding for a new integration. The file is empty and in all scenarios should be renamed + include content, but it leaves a user on a more complete starting point after running `ddev create`

### Additional Notes
<!-- Anything else we should know when reviewing? -->
With the current logic, you can't include an empty directory in a template, so I went with creating an empty file in the new `dashboards` folder. 

Its still required to include this in the manifest.json file, but since the file name should change, I left the manifest as is. 

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have `changelog/` and `integration/` labels attached

OneHunnid commented:
Does this also include an /images folder? Partners will need to add an image anyways for their paid app, so they'll have to add their own images folder regardless. Seems like another quick win for everyone.

nmuesch commented:
Thanks for the reviews.
@ChristineTChen I changed the name of the example dashboard files, I used `{check_name}` as its the normalized version of `{integration_name}`. 

@OneHunnid I added in an empty images folder (I had to include an empty `.gitkeep` in order to have these empty directories tracked by git.
